### DISCUSSION TITLE
Adding shadcn to my nuxt project using layers

### DISCUSSION THREAD

**Discussion by torbentschechne** [unovue/shadcn-vue#970] [Q&A]
Hey guys,

I followed the installation to add the shadcn nuxt module to my project.

I have a structure like this:

```
frontend/apps/app
frontend/packages/core
```

The `packages/core` stuff is an layer that provides all components etc that the apps will use commonly. I added the nuxt module while being in the path `frontend/packages/core`. The docs now state that I should run this command: `npx shadcn-vue@latest init`

Now I wonder, where should I run this command? In the `frontend/packages/core` or in the `root` or in `frontend/apps/app`. My assumption would be to run it in the `frontend/packages/core` as here I've added the module. Is that correct?

Edit: And another question, my shadcn config object in the nuxt config get's this error:

```
Vue: Object literal may only specify known properties, and shadcn does not exist in type InputConfig<NuxtConfig, ConfigLayerMeta>
```

**Comment by memic84** [comment]
Hey @torbentschechne 

We are also using layers a lot, but haven't been able to get this to work with tailwind v4 yet.

It depends on your setup, as i have not all the information but if you have installed Nuxt in the packages/core and install the module also there, where you keep your component then you should also run the command from there. 

Regarding the error, did you run nuxt prepare?
